upload/src/addons/SV/WarningImprovements/_output/templates/public/sv_member_warn_chooser.html

```diff
@@ -5,33 +5,47 @@
 <xf:js addon="SV/WarningImprovements" src="sv/chosen/chosen.jquery.js" min="1" /><xf:comment>missing?</xf:comment>
 <xf:js addon="SV/WarningImprovements" src="sv/warningimprovements/warnings.js" min="1" />
 
-<div id="warningViewSelect" class="JsOnly" style="display: none;">
-	<xf:formrow label="{{ phrase('warning_type') }}">
-		<xf:select name="warning_definition_id" value="{$xf.visitor.style_id}" label="{{ phrase('style') }}" class="warningItems WarningSelector"
-				   data-no-results-text="{{ phrase('sv_no_results_match') }}"
-				   data-place-holder-text="{{ phrase('sv_choose_a_warning') }}">
-			<xf:option value=""></xf:option>
-			<xf:optgroup label="{$warningCategory.title}">
-				<xf:foreach loop="$warningCategory.warnings" value="$warning">
-					<xf:option class="warningDefinition" value="{$warning.warning_definition_id}">{$warning.title}</xf:option>
+<div id="warningViewRadio" style="display: none;">
+	<xf:selectrow name="warning_definition_id" value="0" label="{{ phrase('warning_type') }}" class="warningItems WarningSelector" data-no-results-text="{{ phrase('sv_no_results_match') }}" data-place-holder-text="{{ phrase('sv_choose_a_warning') }}">
```

Comment:
This is unfortunately a major change from how XF1 worked in layout.

For example;
```
General Warnings
  - Serious Warnings
     - Warning 1
     - Warning 2
     - Warning 3
  - Minor Warnings
     - Warning 4
     - Warning 5
     - Warning 6
  - Warning 7
Subforum Warnings
  - Subforum Warning 1
  - Subforum Warning 2
  - Subforum Warning 3
```

There should be a section label "General Warnings", followed by the sub-category and warnings in a chunk of a radio boxes.

Then a section labeled "Subforum Warnings" with it's warnings.